kube/services/jobs/cedar-ingestion-job.yaml

```diff
@@ -84,128 +64,88 @@ spec:
-            let count=0
-            while [[ ! -f /mnt/shared/access_token.txt && $count -lt 50 ]]; do
-              echo "Waiting for /mnt/shared/access_token.txt";
-              sleep 2
-              let count=$count+1
-            done
+
             pip install pydash
             export GEN3_HOME="$HOME/cloud-automation"
-            export ACCESS_TOKEN="$(cat /mnt/shared/access_token.txt)"
-            python ${GEN3_HOME}/files/scripts/healdata/heal-cedar-data-ingest.py --access_token $ACCESS_TOKEN --directory $CEDAR_DIRECTORY_ID --hostname $HOSTNAME
+            python ${GEN3_HOME}/files/scripts/healdata/heal-cedar-data-ingest.py --directory $CEDAR_DIRECTORY_ID --client_id $CEDAR_CLIENT_ID --client_secret $CEDAR_CLIENT_SECRET --hostname $HOSTNAME
             echo "All done - exit status $?"
-      - name: fence
-        GEN3_FENCE_IMAGE
-        imagePullPolicy: Always
-        env:
-          - name: PYTHONPATH
-            value: /var/www/fence
-          - name: SUBMISSION_USER
-            GEN3_SUBMISSION_USER|-value: "cdis.autotest@gmail.com"-|
-          - name: TOKEN_EXPIRATION
-            value: "3600"
-          - name: FENCE_PUBLIC_CONFIG
-            valueFrom:
-              configMapKeyRef:
-                name: manifest-fence
-                key: fence-config-public.yaml
-                optional: true
-        volumeMounts:
-# -----------------------------------------------------------------------------
-# DEPRECATED! Remove when all commons are no longer using local_settings.py
-#             for fence.
-# -----------------------------------------------------------------------------
-          - name: "old-config-volume"
-            readOnly: true
-            mountPath: "/var/www/fence/local_settings.py"
-            subPath: local_settings.py
-          - name: "creds-volume"
-            readOnly: true
-            mountPath: "/var/www/fence/creds.json"
-            subPath: creds.json
-          - name: "config-helper"
-            readOnly: true
-            mountPath: "/var/www/fence/config_helper.py"
-            subPath: config_helper.py
-          - name: "json-secret-volume"
-            readOnly: true
-            mountPath: "/var/www/fence/fence_credentials.json"
-            subPath: fence_credentials.json
-# -----------------------------------------------------------------------------
-          - name: "config-volume"
-            readOnly: true
-            mountPath: "/var/www/fence/fence-config-secret.yaml"
-            subPath: fence-config.yaml
-          - name: "yaml-merge"
-            readOnly: true
-            mountPath: "/var/www/fence/yaml_merge.py"
-            subPath: yaml_merge.py
-          - name: "fence-jwt-keys"
-            readOnly: true
-            mountPath: "/fence/jwt-keys.tar"
-            subPath: "jwt-keys.tar"
-          - name: shared-data
-            mountPath: /mnt/shared
-        command: ["/bin/bash" ]
-        args:
+            touch /mnt/shared/success
```

Comment:
this file will be touched even if the previous python script has exited with a non-0 status?

Reply:
Doh! Thank you for catching that! Updated.